Makefile

```diff
@@ -1,6 +1,7 @@
 TEST_TIMEOUT?=5m
-REPORT?=report.xml
+REPORT?=outputs/godog-report.out
```

Comment:
Can we ignore this file too? I'd also use the `test-reports` directory to create it

Reply:
And keeping the XML format, so that the Jenkinsfile uses common patterns when archiving the test artefacts

Reply:
I'll in fact remove it